pkg/controller/cincinnati/new.go

```diff
@@ -301,6 +344,27 @@ func (k *kubeResources) newDeployment(instance *cv1alpha1.Cincinnati) *appsv1.De
 			},
 		},
 	}
+	if k.UseInitContainer(instance) {
+		dep.Spec.Template.Spec.InitContainers = []corev1.Container{
+			*k.graphDataInitContainer,
+		}
+	}
+
+	return dep
+}
+
+func (k *kubeResources) newGraphDataInitContainer(instance *cv1alpha1.Cincinnati, image string) *corev1.Container {
+	return &corev1.Container{
+		Name:            NameInitContainerGraphData,
+		Image:           image,
+		ImagePullPolicy: corev1.PullIfNotPresent,
+		VolumeMounts: []corev1.VolumeMount{
+			corev1.VolumeMount{
+				Name:      "cincinnati-graph-data",
+				MountPath: "/tmp/cincinnati/graph-data",
```

Comment:
I doubt this matters in the case of matters (that /tmp doesn't survive reboot), but I wonder if we should be using a non `tmp` directory for the graph-data?

Reply:
I had the same thought. I'm a fan of using the [FHS](http://refspecs.linuxfoundation.org/fhs.shtml) as intended even in containers. The only reason I didn't bring it up is that we got this value from the [templates in cincinnati's own repo](https://github.com/openshift/cincinnati/blob/c1e2b4e/dist/openshift/cincinnati.yaml#L329). :shrug: I could go either way on it. Since it's a volume formally being mounted in, something in `/var/` could be perfectly reasonable.

Reply:
It is the VolumeMount that grabs my attention. `/var/` sounds good to me.